### ISSUE TITLE
Why PSNR is very little?

### ISSUE THREAD

**Issue by soaringleefighting** [xiph/rav1e#2785]
Recently I test rav1e, but I have two questions following:
1. qp range is from 0 to 255 for rav1e?
2. The output PSNR is very little(about 10), but PSNR of other codec is about 35.

**Comment by BlueSwordM** [comment]
1. Yes, the quantizer(quality range, it's not a pure quantizer in rav1e) is from 0-255.

2. What do you mean by that?

**Comment by soaringleefighting** [comment]
I meant that the PSNR value generated by rav1e is very small, for example the PSNR is only 10 but quantizer is  set to zero. I don't understand this.

**Comment by kornelski** [comment]
With such low PSNR you should be able to see yourself what went wrong. If the encoded frames look fine, then maybe it's a measurement error? For example, are you comparing correct color spaces? Same chroma subsampling?

**Comment by jnnks** [comment]
I had similar experiences. ffmpeg for example calculates values which are a lot (x2.5) higher.

**Comment by kornelski** [comment]
I shall add that PSNR is an unreliable metric of visual quality. 

Modern video encoders aim for perceptual quality, not PSNR, so thy may deliberately make PSNR worse in order to make video look better to humans.

Metrics like SSIM and VMAF are less bad than PSNR.

**Comment by jnnks** [comment]
Fair point. PSNR is so unreliable, because it's such a simple calculation.
That also means that it's concerning to see different scores for the same videos with two different tools (rav1e built-in and ffmpeg).
